src/register/messages.jsx

```diff
@@ -123,7 +123,7 @@ const messages = defineMessages({
   },
   'support.education.research': {
     id: 'support.education.research',
-    defaultMessage: 'Support education research by providing additional information. (Optional)',
+    defaultMessage: 'Support education research by providing additional information. (optional)',
```

Comment:
Shouldn’t we keep it capital here? It's after the dot.

Reply:
good catch. will confirm from @wajeeha-khalid  to be sure